src/components/Sections/ItemsSection.js

```diff
@@ -143,6 +162,8 @@ class ItemsSection extends Component {
                 <SectionMarkdown text={String(item.data)} />;
                 if (type === SECTION_ITEM_TYPE.html) {
                   dataDisplay = <SectionHTML text={item.data} />;
+                } else if (type === SECTION_ITEM_TYPE.tagsSelect) {
```

Comment:
we can perhaps add multiselect here as well

Reply:
There are a lot of things to style :).
 maybe in one of the next PRs (need to support GOM reports first)